.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,7 +1,7 @@
 ---
 name: Feature request
 about: Suggest an idea for this project
-
+labels: Refinement Needed
```

Comment:
Are we sure this doesn't need quotes, as its 2 words?

Reply:
Apparently it's only required if the label has special characters or emojis.

Reply:
Ah Yep, [it uses commas](https://github.community/t5/How-to-use-Git-and-GitHub/Apply-labels-automatically-at-issue-creation/m-p/15909/highlight/true#M4903)